Component resource Option (#109)
Leafwing-Studios/leafwing_input_manager: RELEASES.md

### Enhancements

> - users are now able to use the `ActionState` Component as a resource externally to an entity

Reviewer: ```suggestion
- if desired, users are now able to use the `ActionState` and `InputMap` structs as standalone resources
```
